Hashes/MD5.js

```diff
@@ -0,0 +1,221 @@
+//= ===============================================================
```

Comment:
This line and the following is just spam.

Reply:
Spam has been removed.